<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>381</width>
      <height>192</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Next day</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Duration</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Load dates</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>210</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>210</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Birthday</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>552</x>
      <y>260</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>confirm birthday</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>430</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev day</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>430</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Days till your birthday</string>
    </property>
   </widget>
   <widget class="QLabel" name="prevDayLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev day</string>
    </property>
   </widget>
   <widget class="QLabel" name="birthdayDurationLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>470</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Days</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>390</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
